ragoon/__init__.py

```diff
@@ -57,7 +61,10 @@ def __init__(
         # Embedding texts into db
         self.embedder = None
         if embedder is None and self.config.embed is not None:
-            self.embedder = ChromaEmbedder(self.config)
+            if self.config.embed.model == "jxm/cde-small-v1":
```

Comment:
@gkaretka I really don't like this hardcoding too much -- could we have a specific config variable that would choose between ChromaEMbedder and CDEEmbedder?